RDFSharpSpecflowTests/Features/Queries.feature

```diff
@@ -0,0 +1,49 @@
+﻿Feature: Queries
+
+Testing different queries on the RDFSharp project:
+- find max 3 values
+- calculating average
+- determin max value
+
+Scenario Outline: Executing a query to get the phones with the longest battery life
+	Given a graph with <numOfPhones> phones and their battery capacity
+	And a query that selects the top <numOfPhones> phones with the longest battery life based on battery capacity
+	When executing the created query on the graph
+	Then the result, query and graph objects are not null and result contains <numOfPhones> objects
+	And receiving the actual top <numOfPhones> phones with the longest battery life
+
+	Examples:
```

Comment:
Ezek határértékek?

Reply:
Nem. Viszont amennyiben a dokumentációban szerepelnénk határértekek ezeket a paramétereket illetően, akkor tényleg azokra kéne tesztelni. Ezekben a tesztekben megpróbáltam a Scenario Outlineok sajátosságát kihasználni, hogy egyszerre több tesztet is keszíthetek, más paraméterekkel, kipróbálva, hogy pl.: mi történik, ha kisebb és sokkal nagyobb számú elemet adok a gráfhoz.